docs/src/docs/examples.clj

```diff
@@ -257,6 +257,40 @@ node)
 ;; end::query-with-arguments5-r[]
 )
 
+(defn query-example-with-predicate-1 [node]
+  (crux/q
+   (crux/db node)
+   ;; tag::query-with-pred-1[]
+   {:find '[age]
+    :where '[[(odd? age)]]
+    :args [{'age 22} {'age 21}]}
+   ;; end::query-with-pred-1[]
+ ))
+
+#_(comment
+;; tag::query-with-pred-1-r[]
+#{[21]}
+;; end::query-with-pred-1-r[]
+)
+
+(defn query-example-lazy [node]
+  ;; tag::lazy-query[]
+  (crux/q
+   (crux/db node)
+   (crux.api/new-snapshot (crux/db node))
```

Comment:
I think we need to show use of a `with-open` in this example otherwise the snapshot will not get closed.